### DISCUSSION TITLE
Future templates for Quarto Manuscripts

### DISCUSSION THREAD

**Discussion by 0ceangypsy** [quarto-dev/quarto-cli#9028] [Feature Requests]
### Description

Dear Quarto Team, 

first and foremost, thank you for all the good work that you have been doing. It's greatly appreciated. I would like to ask the roadmap for the future journal-templates that are planned to work with quarto-manuscripts. 

I would like to make a request for the following templates: 
- Academy of Management Journals 
- Springer Nature journals 
- INFORMS journals

An alternative idea could be to enable a LaTeX template from the user's end either from existing local directory or online (e.g. Overleaf) to make templates more sustainable, if it is technically possible.

**Comment by mcanouil** [comment]
I believe you can use any formats extensions in a Manuscript project.
As many users from the community, you can create one and use it.
- https://quarto.org/docs/extensions/listing-journals.html

For instance "Springer Nature journal" already has a community based extension: https://github.com/christopherkenny/nature

It's unlikely Quarto team will make new templates and maintain them while there are plenty of work on Quarto CLI and that there is an extension mechanism.

**Reply by dmi3kno** [reply]
I just wanted to leave a comment here that INFORMS has multiple templates stored [here](https://pubsonline.informs.org/authorportal/latex-style-files)

**Reply by 0ceangypsy** [reply]
Thank you this is great to know
